apps/authentication-service/src/shared/utils.test.ts

```diff
@@ -0,0 +1,67 @@
+import { Request } from "express";
+
+import { getClientTokenFromRequest, parseResponse } from "@/shared/utils";
+
+describe("utils", () => {
+  describe("parseResponse", () => {
+    it("parseResponse when response is ok, and body is null", async () => {
+      const response = {
+        ok: true,
+        status: 200,
+        statusText: "OK",
+        json: jest.fn(),
+        body: null,
+        headers: {},
+      } as unknown as Response;
+
+      const result = await parseResponse(response);
+
+      expect(result).toEqual(response);
+    });
+
+    it("parseResponse when response is ok, and body is not null", async () => {
```

Comment:
these it statements don't tell me what this test should be testing. It tells me the case, but not what should happen.

it("returns the json body if the response is ok and there is a body")